js/connection.js

```diff
@@ -151,21 +151,33 @@ weechat.factory('connection',
                     _requestHotlist().then(function(hotlist) {
                         handlers.handleHotlistInfo(hotlist);
 
-                        if (successCallback) {
-                            successCallback();
-                        }
                     });
+                    // Schedule hotlist syncing every so often so that this
+                    // client will have unread counts (mostly) in sync with
+                    // other clients or terminal usage directly.
+                    setInterval(function() {
+                        if ($rootScope.connected) {
+                            _requestHotlist().then(function(hotlist) {
```

Comment:
Hm we used to just switch buffers in weechat so I tried to mimic the old behaviour but either is fine by me